solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/DesignSystem/Constants.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+struct Constants {
+    static let preview_square_size = 400.0
```

Comment:
**[SUGGESTION]**
Use camelCase instead, it's more common in the Swift world.
```suggestion
    static let previewSquareSize: CGFloat= 400.0
```

Reply:
👍